calyx-py/calyx/py_ast.py

```diff
@@ -353,7 +353,7 @@ def doc(self) -> str:
 @dataclass
 class Empty(Control):
     def doc(self) -> str:
-        return "empty"
+        return ""
```

Comment:
Just double-checking that this is correct? Like, is it allowed to do `seq { ; ; ; }`? It probably is, as of #1266, right?

Reply:
Well, that is probably not parsable. Something like `@sync; @foo; @bar` is. Regardless, `empty` is not a Calyx keyword

Reply:
I see; got it!